docs/Reference/Pantheon-CLI-Syntax.md

```diff
@@ -508,43 +477,48 @@ The default is 30303.
 !!!note
     This option is not used when running Pantheon from the [Docker image](../Getting-Started/Run-Docker-Image.md#exposing-ports). 
 
-### rinkeby
+### genesis-file
+
+Genesis file is used to create a custom network.
+
+!!!tip
+    If you need to use one of the well-known networks like Ethereum MainNet, you should use
+    the [`--network`](#network) option instead as all nodes of a same network have to define the 
+    exact same genesis parameters.
 
-!!!important
-    This option is deprecated in favor of the new `--network` option.
-    It will be completely removed in the 0.9 release.
-    
 ```bash tab="Syntax"
---rinkeby
+--genesis-file=<FILE>
+```
+
+```bash tab="Example Command Line"
+--genesis-file=/home/me/me_node/customGenesisFile.json
 ```
 
 ```bash tab="Example Configuration File"
-rinkeby=true
+genesis-file="/home/me/me_node/customGenesisFile.json"
 ```
 
-Uses the Rinkeby test network.
-Default is `false`.
-  
-  
-### ropsten
+The path to the genesis file. There's no default for this option.
```

Comment:
Suggest removing 'There's no default for this option' -> I don't think it's needed. They'll get an error if they try and use it without specifying anything.